### PR TITLE
Upgrade to go1.19

### PR DESCRIPTION
**Pull request by aminesnow**
### Description

Update to use Go 1.19.

/cc. 
/assign 

### Links

- JIRA: https://issues.redhat.com/browse/LOG-3150

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/elasticsearch-proxy
 
-go 1.18
+go 1.19
```
Comment (periklis) [inline go.mod:3]: No changes in `go.sum` with `go mod tidy` after this?

Comment (aminesnow) [reply go.mod:3]: nope

**Review by periklis — APPROVED** [review]

**Pull request merged**